attest/application_key.go

```diff
@@ -101,3 +102,8 @@ func (k *Key) Marshal() ([]byte, error) {
 func (k *Key) CertificationParameters() CertificationParameters {
 	return k.key.certificationParameters()
 }
+
+// Blobs returns private and public blobs to be used by tpm2.Load().
+func (k *Key) Blobs() ([]byte, []byte, error) {
```

Comment:
Blah, sorry I didn't catch this. But please name the return values. 

func (k *Key) Blobs() (pub, priv []byte, err error)

Reply:
Done